admin/installer/setup_volumehub.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Set up Volume Hub if a token has been supplied.
+set -ex
+
+: ${volumehub_token:?}
+: ${node_number:?}
+
+if test -z "${volumehub_token}"; then
+    # No Volume Hub token supplied.
```

Comment:
That part was broken actually.
If you omitted the volumehub_token then the entire (combined) script would exit and the final wait condition callback didn't happen.

I got rid of that block.